tests/cypress/e2e/api-spec.cy.js

```diff
@@ -0,0 +1,62 @@
+            headers: {
+                'Content-type': 'application/json; charset=UTF-8',
+            },
+        }).then((response) => {
+            expect(response.status).to.eq(200);
+            expect(response.body).to.include({"status": true});
+        });
+    });
+
+    it('downloading a csv of the report data should succeed', () => {
```

Comment:
Since the test starts with "it", should this test be "should succeed in the downloading  of a the report data"?